src/rgw/rgw_admin.cc

```diff
@@ -324,6 +324,8 @@ enum {
   OPT_BUCKET_SYNC_STATUS,
```

Comment:
the bucket enable/disable cmd should be added into function "_usage" in rgw_admin.cc？